Bottom: 416e919f8a74ef33f546d2870b71d026f8ca4b19
Top:    81fd77c2dbde70c4b2d73d1b7102dd45f04d16b4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:55:35 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index f97d90f..e3ba2a8 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -326,13 +326,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
